src/analysis/graphs.md

```diff
@@ -44,55 +48,59 @@ outputs the custom graph in JSON format.
 
 Here's a short description for every output format available:
 
-### Ascii Art ** (e.g. `agf`)
+### Ascii Art (e.g. `agf`)
 
 Displays the graph directly to stdout using ASCII art to represent blocks and edges.
 
-_Warning: displaying large graphs directly to stdout might prove to be computationally expensive and will make rizin not responsive for some time. In case of doubt, prefer using the interactive view (explained below)._
+_Warning: displaying large graphs directly to stdout might prove to be computationally expensive and will make Rizin
+not responsive for some time. In case of doubt, prefer using the interactive view (explained below)._
 
 ### Interactive Ascii Art (e.g. `agfv`)
 
 Displays the ASCII graph in an interactive view similar to `VV` which allows to move the screen, zoom in / zoom out, ...
 
 ### Tiny Ascii Art (e.g. `agft`)
 
-Displays the ASCII graph directly to stdout in tiny mode (which is the same as reaching the maximum zoom out level in the interactive view).
+Displays the ASCII graph directly to stdout in tiny mode (which is the same as reaching the maximum zoom out level
+in the interactive view).
 
-### Graphviz dot	(e.g. `agfd`)
+### Graphviz dot (e.g. `agfd`)
 
-Prints the dot source code representing the graph, which can be interpreted by programs such as [graphviz](https://graphviz.gitlab.io/download/) or online viewers like [this](http://www.webgraphviz.com/)
+Prints the dot source code representing the graph, which can be interpreted by programs such as
+[graphviz](https://graphviz.gitlab.io/download/) or online viewers like [webgraphviz](http://www.webgraphviz.com/).
 
-### JSON	(e.g. `agfj`)
+### JSON (e.g. `agfj`)
```

Comment:
these commands are outdated, please check how the command `agf` is executed and update the reference.